### PR TITLE
Add Infino to partners page

### PR DESCRIPTION
**Pull request by vinaykakade**
### Description
Add Infino to partners page. Infino offers a schemaless, serverless managed OpenSearch service. We have replaced Lucene with a next-gen proprietary engine optimized for Observability use cases and internal developer platforms. Infino is planning to launch at OpenSearchCon Berlin.

Verified that the partners page as well as the partner detail page for Infino are displayed correctly on a local version of OpenSearch project website.
 
### Issues Resolved
Add Infino to partners page. #2714 

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pajuric** [comment]
@vinaykakade - If you have time to also [create a partner page for the site by filing this issue](https://github.com/opensearch-project/project-website/issues/new?assignees=&labels=enhancement%2C+untriaged&projects=&template=partner_information.md&title=), I would appreciate it. This was we can include you in the partner selector guide that is going live in the next few weeks.

**Comment by ekechinwokah** [comment]
> @vinaykakade - If you have time to also [create a partner page for the site by filing this issue](https://github.com/opensearch-project/project-website/issues/new?assignees=&labels=enhancement%2C+untriaged&projects=&template=partner_information.md&title=), I would appreciate it. This was we can include you in the partner selector guide that is going live in the next few weeks.

done

File: _partners/infino.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+region: 'North America, South America, Europe, Middle East, Africa, Asia Pacific, Australia'
+contact: 'tech@infino.ai'
+opensearch_tech: 'Observability'
+industries: 'Software and Technology'
+main_office_location:  |
+   Infino AI Inc.
+   19266 Coastal Hwy
+   Unit 4-250
+   Rehoboth Beach, DE 19971
+products:
```
Comment (nateynateynate) [inline _partners/infino.md:22]: I think this line might be repeated on accident.

Comment (nateynateynate) [reply _partners/infino.md:22]: @vinaykakade : I don't think I have access to push a quick change to your `main` branch - feel free to revise your PR when you get a chance and we'll push you live!

**Review by nateynateynate — CHANGES_REQUESTED** [review]
One quick change if you don't mind.

File: _partners/infino.md (hunk 1 above)
Comment (vinaykakade) [reply _partners/infino.md:22]: Hi @nateynateynate - thanks you for pointing this out. Fixed. Also, have given you push access to my fork.

**Review by nateynateynate — APPROVED** [review]

**Review by nateynateynate — APPROVED** [review]
Welcome to the list of OpenSearch partners!

**Pull request merged**